hive/datadog_checks/hive/data/metrics.yaml

```diff
@@ -384,31 +384,25 @@ jmx_metrics:
       bean: metrics:name=memory.non-heap.init
       attribute:
         Value:
-          alias: hive.server.memory.non-heap.init
-          metric_type: gauge
-  - include:
-      bean: metrics:name=memory.non-heap.usage
```

Comment:
usage one got deleted?

Reply:
Yes, it was removed from `config.default.yaml` and `metadata.csv`, but was still in `metrics.yaml`